src/componentUpdater.ts

```diff
@@ -234,6 +234,7 @@ export function extractReservationListProps(props: AgendaProps) {
     renderDay,
     renderItem,
     renderEmptyDate,
+    ...others
```

Comment:
No. `...others` contains all the props we don't want to pass. By passing `others` you're actually passing ALL the props.